cu-webteam/d8-platform
# updating footer block config and minor updates to other config files

# Description

Update footer contact block and footer badge block config so that the blocks are no longer broken.

## Related PRs

None

## Todos

- [x] Tests _tested locally, no automated tests_
- [ ] Documentation _minor config changes, no documentation required_

## Deploy Notes

None

## Steps to Test or Reproduce

* Pull code down locally
* Start VM
* Run `drush cim` to import config changes into the local site
* Check that the footer contact block now shows in the footer
* Check that the footer badge block now shows in the footer
